sdk/hybridcompute/Microsoft.Azure.Management.HybridCompute/src/Microsoft.Azure.Management.HybridCompute.csproj

```
@@ -0,0 +1,25 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <!-- Please do not move/edit code below this line -->
+  <Import Project="$(RepoEngPath)/mgmt/AzSdk.reference.props" />
+  <!-- Please do not move/edit code below this line -->
+
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.Management.HybridCompute</PackageId>
+    <Description>Provides developers with libraries for managing hybrid machines</Description>
+    <Version>0.0.1</Version>
```

First version should be `0.1.0` please. And I believe your module will be in preview, right?
```suggestion
    <Version>0.1.0-preview</Version>
```

---

yes. updated.